### DISCUSSION TITLE
Diagnostic error in lazy syntax

### DISCUSSION THREAD

**Discussion by semi710** [glacambre/firenvim#1563] [Q&A] [answered]
![image](https://github.com/glacambre/firenvim/assets/60490474/73ecf3d6-1f9e-4e09-9caf-5541ca304e69)
why this error is there and is there any solution?

**Comment by glacambre** [comment, marked as answer]
I don't use/know lazy.nvim, but reading the message `Cannot assign "string" to "(string|LazyPlugin)[]"` makes me think that you're trying to pass a string where an array of strings is expected. Try `plugins = {"firenvim"}` instead.

**Reply by semi710** [reply]
Don't know how but the diagnostic is gone for now.
i changed nothing.
keeping it here for future reference if faced again.
